presto-main/src/main/java/com/facebook/presto/operator/BigintGroupByHash.java

```diff
@@ -34,8 +34,10 @@
 import static com.facebook.presto.type.TypeUtils.NULL_HASH_CODE;
 import static com.facebook.presto.util.HashCollisionsEstimator.estimateNumberOfHashCollisions;
 import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Verify.verify;
```

Comment:
cmt msg "Refactor BigintGroupByHash for yield preparation":

replace "Adapt" with "Apply", "Use .. in" or something